lua/quickmath/init.lua

```diff
@@ -116,6 +120,227 @@ function go_eol()
 
 end
 
+function vec(...)
```

Comment:
Just a small comment, the syntax for vector initialization with `a = vec {1,2,3}` instead of `a = vec(1,2,3)` could be nicer. I'll let you decide because it's strictly design and it's "your" code.

Reply:
Not sure I'm familiar with that type of initialization. How would the signature look for that?

Reply:
It's a small trick in lua. If the function takes a single argument, like a table, the parenthesis for the call can be omitted.

```lua
function myfun(a)
   -- a is a table
   -- do something with a
end

-- normally you would call the function like this
myfun({1,2,3,4})

-- but lua allows this
myfun {1,2,3,4}
```

Works also for strings.

Reply:
I opted for allowing all variants.